lib/simplifyExpression/fractionsSearch/cancelLikeTerms.js

```diff
@@ -20,6 +20,52 @@ class CancelOutStatus {
 // e.g. (2x^2 * 5) / 2x^2 => 5 / 1
 // Returns a Node.Status object
 function cancelLikeTerms(node) {
+  // Cancel out terms if the term being multiplied
```

Comment:
I think this comment isn't super clear about what 'the term being multiplied' is
can you add an example?